pytensor/tensor/nlinalg.py

```diff
@@ -677,41 +690,193 @@ def matrix_power(M, n):
     return result
 
 
-def norm(x, ord):
-    x = as_tensor_variable(x)
+def _multi_svd_norm(x: ptb.TensorVariable, row_axis: int, col_axis: int, op: Callable):
```

Comment:
Maybe a bit more readable?

```suggestion
def _multi_svd_norm(x: ptb.TensorVariable, row_axis: int, col_axis: int, reduce_op: Callable):
```